src/pages/index.js

```diff
@@ -48,35 +53,66 @@ import {
 
 const addCardForm = document.forms["add-card-form"];
 
+const profileAvatarButton = document.querySelector(".profile__avatar-button");
+
+const avatarEditForm = document.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms